eng/_util/cmd/run-builder/run-builder_windows.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//go:build windows
```

Comment:
Inline build tags are normally omitted if the constraint is already covered in the file name.

Reply:
Ah, right, slipped my mind that the filename is actually a constraint (rather than just a naming convention).